modules/op-advantages-and-disadvantages-of-non-versioned-and-versioned-cluster-tasks.adoc

```diff
@@ -0,0 +1,32 @@
+// This module is part of the following assembly:
+//
+// *cicd/pipelines/managing-nonversioned-and-versioned-cluster-tasks.adoc
+:_content-type: CONCEPT
+[id="advantages-and-disadvantages-of-non-versioned-and-versioned-cluster-tasks_{context}"]
+= Advantages and disadvantages of non-versioned and versioned cluster tasks
+
+Before adopting non-versioned or versioned cluster tasks as a standard in production environments, cluster administrators might consider their advantages and disadvantages.
+
+.Advantages and disadvantages of non-versioned and versioned cluster tasks
+[options="header"]
+|===
+
+| Cluster task | Advantages | Disadvantages
+
+| Non-versioned cluster task (NVCT)
+a| 
+* If you prefer deploying pipelines with the latest updates and bug fixes, use the NVCT. 
+* Upgrading the Operator upgrades the non-versioned cluster tasks, which consume fewer resources than multiple versioned cluster tasks.  
+a| If you deploy pipelines that use NVCT, they might break after an Operator upgrade if the automatically upgraded cluster tasks are not backward-compatible. 
+
+| Versioned cluster task (VCT)
+a| 
+* If you prefer stable pipelines in production, use the VCT. 
+* The earlier version is retained on the cluster even after the later version of a cluster task is installed. You can continue using the earlier cluster tasks. 
+a| 
+* If you continue using an earlier version of a cluster task, you might miss the latest features and critical security updates.
+* The earlier versions of cluster tasks that are not operational consume cluster resources.  
+* When upgraded , the Operator cannot manage the earlier VCT. You can delete the earlier VCT manually using the `oc delete clustertask` command, but you cannot restore it.  
```

Comment:
extra space before comma (not sure if it would render correctly)

Reply:
Eagle eyes, as always :) Thanks, Pavol. @sounix000 I will go ahead and merge this PR but please fix this as part of another PR.

Reply:
@ppitonak @Preeticp : Resolving this typo here: https://github.com/openshift/openshift-docs/pull/46294/files#r889220287. Thanks for merging it.